src/functions/googleAnalytics.js

```diff
@@ -0,0 +1,21 @@
+import ReactGA from 'react-ga';
```

Comment:
what's the added value of this file to RectGA.... the functions looks exactly the same

Reply:
removed